CHANGELOG.md

```diff
@@ -67,6 +67,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 <!-- Release links -->
 
-[Unreleased]: https://github.com/cosmos/gaia/compare/v9.0.1...release/v9.0.x
```

Comment:
Please update the PR to leave this link as it's useful. Ofc with v9.0.3 instead of v9.0.1.